tests/templates/kuttl/overrides/21-assert.yaml

```diff
@@ -4,5 +4,5 @@ kind: TestAssert
 timeout: 30
 commands:
 - script: |
-    kubectl -n $NAMESPACE get cm airflow-kubernetes-executor-pod-template -o json | jq -r '.data."airflow_executor_pod_template.yaml"' | yq -e '.spec.containers.[0].resources.limits | select (.cpu == "750m")'
+    kubectl -n $NAMESPACE get cm airflow-kubernetes-executor-pod-template -o json | jq -r '.data."airflow_executor_pod_template.yaml"' | yq -e '.spec.containers.[0].resources.limits | select (.cpu == "2000m")'
```

Comment:
How did this work before? Is the test actually checking stuff? :thinking:
You changed the CPU limit for the webserver, but this assert checks the executor pod template

Reply:
Yes, good catch, that was broken. Have changed it to ensure that the test works: ecbe53c